web/thesauruses/_meta/lists.json

```diff
@@ -8,8 +8,6 @@
       "name_of_ordered_mutable_list": "What is a ordered (not sorted) mutable list called?",
       "create_a_ordered_mutable_list": "Create the list",
       "ordered_mutable_list_start_number": "What number does it start at?",
-      "ordered_mutable_list_can_be_appended": "Can you append to it?",
-      "ordered_mutable_list_can_be_inserted_in_middle": "Can you insert into the middle of it?",
```

Comment:
These I removed because their content is repeated in the presence/absence of the `insert_into_*_of_*` concepts
and I would argue a user would always rather see the implementation than a simple yes.

Reply:
Makes sense to me.